app/models/package_manager/maven_manager/spring_libs.rb

```diff
@@ -0,0 +1,61 @@
+module MavenManager
+  class SpringLibs < PackageManager::Maven
+    REPOSITORY_SOURCE_NAME = 'SpringLibs'
+
+    def self.repository_base
+      "https://repo.spring.io/libs-release-local"
+    end
+
+    def self.project_names
+      get("http://localhost:8080/springLibReleases/all")
+    end
+
+    def self.recent_names
+      get("http://localhost:8080/springLibReleases/recent")
+    end
+
+    def self.load_names(limit = nil)
+      project_names.each { |name| REDIS.sadd("maven-spring-lib-names", name)}
+    end
+
+    def self.package_link(project, version = nil)
+      if version
+        MavenUrl.from_name(project.name, repository_base).jar(version)
+      else 
+        MavenUrl.from_name(project.name, repository_base).base
+      end
+    end
+
+    def self.download_url(name, version = nil)
+      if version
+        MavenUrl.from_name(name, repository_base).jar(version)
+      else 
+        MavenUrl.from_name(name, repository_base).base
+      end
+    end
+
+    def self.check_status_url(project)
+      MavenUrl.from_name(project.name, repository_base).base
+    end
+
+    def self.versions(_project, name)
+      versions_page = get_raw(MavenUrl.from_name(name, repository_base).base)
+      link_regex = /([\w\-.]+)\/<\/a>/
+      found_versions = versions_page
```

Comment:
I need to take another look through this. There isn't an API or anything, so this looks at all the link names available when you look at the package on the repository directory page and takes the version names from those.

Pseudo code is:
1. load page
2. look for any lines that have a `href` in them.
3. use regex to get text used in the link tag
4. remove the parent directory link from the group, `..`